src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/DefendantResponseController.java

```diff
@@ -43,6 +44,7 @@ public Claim save(
         @PathVariable("externalId") String externalId,
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorization
     ) {
+        response.setResponseMethod(ResponseMethod.DIGITAL);
```

Comment:
use to builder instead

Reply:
This would make the code a lot more complicated as the abstract cannot have a builder so we would need to cast the class appropriately.